jpa/pom.xml

```diff
@@ -23,25 +23,45 @@
         <groupId>jakarta.tck</groupId>
         <artifactId>project</artifactId>
         <version>10.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
     </parent>
-    <artifactId>jpa-tck</artifactId>
-    <name>JPA</name>
-    <description>JPA</description>
+    <artifactId>persistence-tck</artifactId>
+    <version>3.2.0</version>
+    <packaging>pom</packaging>
+    <name>persistence</name>
+    <description>persistence tck tests</description>
+    <modules>
+        <module>common</module>
+        <module>dist</module>
```

Comment:
`dist` looks missing in the PR

Reply:
thanks @lukasj, `git status` was not showing the folder under untracked files, I have added folder using `git add -f`

Reply:
I recommend renaming the folder to `distribution` (or whatever you find suitable) instead - `dist` is often being put to ignore list of VCS (because of `ant`) and one can easily get surprised by local changes not being part of the PR (they are easy to miss)